Source/WebCore/page/Quirks.cpp

```diff
@@ -1678,6 +1677,7 @@ static AccessibilityRole accessibilityRole(const Element& element)
 }
 
 // walmart.com: rdar://123734840
+// live.outlook.com: rdar://152277211
 bool Quirks::shouldIgnoreContentObservationForClick(const Node& targetNode) const
```

Comment:
This now applies to two different domains with different targets and attributes. The code needs to isolate the scope per domain to avoid name clashes. 

(Non-Reviewer status review) 
I let @brentfulgham chime in for a full review or @whsieh

Reply:
To avoid this you can do something like:
https://github.com/WebKit/WebKit/blob/6c66a43574f55efadbca8e69b88161d541b36d10/Source/WebCore/page/Quirks.cpp#L983-L1000

Reply:
Maybe better if we have two OutLook Quirks. I can refactor later if you want. 
https://github.com/WebKit/WebKit/blob/6c66a43574f55efadbca8e69b88161d541b36d10/Source/WebCore/page/Quirks.cpp#L832C1-L842